packages/devops/scripts/ci/validateBranchName.sh

```diff
@@ -22,6 +23,17 @@ function get_branch_name() {
     echo $branch_name
 }
 
+function check_name_is_not_reserved() {
+    local $branch_name=$1
+
+    for reserved_name in ${RESERVED_NAMES[@]}; do
+        if [[ "$branch_name" == "$reserved_name" ]]; then
+            log_error "❌ Branch name is reserved"
```

Comment:
Might as well include `$branch_name` in the log

Reply:
Good call! Certainly won't harm